guacamole/src/main/frontend/src/translations/ca.json

```diff
@@ -127,21 +140,28 @@
         "NAME_MOUSE_MODE_ABSOLUTE" : "Pantalla tàctil",
         "NAME_MOUSE_MODE_RELATIVE" : "Touchpad",
 
-        "SECTION_HEADER_CLIPBOARD"      : "Portapapers",
-        "SECTION_HEADER_DEVICES"        : "Dispositius",
-        "SECTION_HEADER_DISPLAY"        : "Pantalla",
-        "SECTION_HEADER_FILE_TRANSFERS" : "Transferències de fitxers",
-        "SECTION_HEADER_INPUT_METHOD"   : "Mètode d'entrada",
-        "SECTION_HEADER_MOUSE_MODE"     : "Mode d'emulació del ratolí",
+        "SECTION_HEADER_CLIENT_MESSAGES" : "Missatges",
+        "SECTION_HEADER_CLIPBOARD"       : "Porta-retalls",
+        "SECTION_HEADER_DEVICES"         : "Dispositius",
+        "SECTION_HEADER_DISPLAY"         : "Pantalla",
+        "SECTION_HEADER_FILE_TRANSFERS"  : "Transferències de fitxers",
+        "SECTION_HEADER_INPUT_METHOD"    : "Mètode d'entrada",
+        
+        "SECTION_HEADER_MOUSE_MODE"      : "Mode d'emulació del ratolí",
 
+        "TEXT_ANONYMOUS_USER_JOINED"      : "Un usuari anònim s'ha afegit a la connexió.",
+        "TEXT_ANONYMOUS_USER_LEFT"        : "Un usuari anònim ha sortit de la connexió.",
         "TEXT_ZOOM_AUTO_FIT"              : "S'adapta automàticament a la finestra del navegador",
         "TEXT_CLIENT_STATUS_IDLE"         : "Ociós.",
         "TEXT_CLIENT_STATUS_CONNECTING"   : "S'està connectant a Guacamole ...",
         "TEXT_CLIENT_STATUS_DISCONNECTED" : "Heu estat desconnectats.",
         "TEXT_CLIENT_STATUS_UNSTABLE"     : "La connexió de xarxa al servidor Guacamole sembla inestable.",
         "TEXT_CLIENT_STATUS_WAITING"      : "Connectat a Guacamole. Esperant resposta ...",
+        "TEXT_USER_JOINED"                : "{USERNAME} s'ha afegit a la connexió.",
+        "TEXT_USER_LEFT"                  : "{USERNAME} ha sortit de la connexió.",
         "TEXT_RECONNECT_COUNTDOWN"        : "Re-conectant en {REMAINING} {REMAINING, plural, one{segon} other{segons}}...",
         "TEXT_FILE_TRANSFER_PROGRESS"     : "{PROGRESS} {UNIT, select, b{B} kb{KB} mb{MB} gb{GB} other{}}",
+        "TEXT_CLIPBOARD_AWAITING_FOCUS"   : "Feu clic per veure les dades del porta-retalls...",
```

Comment:
This section should be organized in alphabetical order. I know some of it was already out of order in the original file, but seems like a good opportunity to reorganize it.

Reply:
Organized